polaris-react/src/components/IndexFilters/IndexFilters.tsx

```diff
@@ -263,9 +263,20 @@ export function IndexFilters({
     beginEdit();
   }
 
-  const searchFilterTooltip =
-    filteringAccessibilityTooltip ||
-    i18n.translate('Polaris.IndexFilters.searchFilterTooltip');
+  const searchFilterTooltip = useMemo(() => {
+    const isMac =
+      typeof window === 'undefined'
+        ? false
+        : window.navigator?.platform?.includes('Mac');
+
+    return (
+      filteringAccessibilityTooltip ||
+      `${i18n.translate('Polaris.IndexFilters.searchFilterTooltip')} (${
```

Comment:
Just want to make sure we are sure there's no downside to interpolating the string in this way, as opposed to using the variable replacement that's built into the `i18n` library, so like:

```tsx
i18n.translate('Polaris.IndexFilters.searchFilterTooltip', { shortcut: isMac ? '⌘F' : 'F' });
```
if the JSON string is:

```
searchFilterTooltip: "Search and filter {shortcut}"
```

Reply:
Also a heads up that this will have to be updated [here](https://github.com/Shopify/web/blob/main/app/shared/index-filters/components/IndexFiltering/IndexFiltering.tsx#L102) too, as we override the tooltip to include the context around which index we're currently searching/filtering when using in admin.